OpenQA/Isotovideo/Runner.pm

```diff
@@ -235,6 +246,23 @@ sub _init_bmwqemu ($, @args) {
     }
 }
 
+sub exit_code_from_test_results ($self) {
```

Comment:
I guess for this particular package it is *ok* that the variable `$self` is not actually used. (So no change requested here. I just find it a bit unusual.)